### DISCUSSION TITLE
rx6650 xt support on ventura/sonoma

### DISCUSSION THREAD

**Discussion by scanduz** [ChefKissInc/NootRX#52] [Q&A] [answered]
Hi, I ask for support on installing RX 6650 XT on Ventura or Sonoma. Thanks to nootrx I got it working on monterey but I would like to extend it working on ventura or sonoma. I got flashing screens before logging into sonoma, I tried all the solutions but nothing solved the problem. Sorry but I wouldn't allow myself to disturb you, but I'm desperate and I can't understand where I'm wrong. This procedure was performed on an Apple Mac Pro 5.1

**Comment by june5910aaaa** [comment]
my 6650xt works well on sonoma14.0,do not know what happened for you.
maybe you can use whatevergreen to hack in the sys,then change to nootrx

**Comment by scanduz** [comment]
I'm using a real Mac Pro 5.1, I can run the installation but at the login screen the monitor goes black and starts flashing even occasionally, with Monterey everything works normally. Can you share the kext you use that I test on mine? Thanks for your kind response

**Comment by june5910aaaa** [comment]
i am just use the newest version

________________________________
发件人: scanduz ***@***.***>
发送时间: 2024年5月8日 16:02
收件人: ChefKissInc/NootRX ***@***.***>
抄送: june5910aaaa ***@***.***>; Comment ***@***.***>
主题: Re: [ChefKissInc/NootRX] rx6650 xt support on ventura/sonoma (Discussion #52)


I'm using a real Mac Pro 5.1, I can run the installation but at the login screen the monitor goes black and starts flashing even occasionally, with Monterey everything works normally. Can you share the kext you use that I test on mine? Thanks for your kind response

―

**Reply by scanduz** [reply]
1.0.0?

**Reply by scanduz** [reply]
this is the version I'm using 1.0.0

**Reply by scanduz** [reply]
tomorrow I'll try to add whatevergreen to the kext too,

**Reply by june5910aaaa** [reply]
![截屏2024-05-10 11 10 04](https://github.com/ChefKissInc/NootRX/assets/19283240/cfb87a18-40bb-49f6-a544-cc29f54e3ced)
u can download it from cl

**Comment by scanduz** [comment]
Thank you very much for the support, I download the fix version 6570/6750 correct? E I use the release versionScanduzIl giorno 10 mag 2024, alle ore 05:11, june5910aaaa ***@***.***> ha scritto:﻿
2024-05-10.11.10.04.png (view on web)
u can download it from cl

**Reply by june5910aaaa** [reply]
yes，it is，did you make your 6650xt worked?

**Comment by scanduz** [comment]
no unfortunately my mac pro 5.1 supports the rx 6650 xt only on monterey. I inserted the updated nootrx kext but got the flashing screen, then I tried to insert the whatevergreen kext without the boot args but flashing black screen in sonoma. I don't understand what's wrong. To declare kexts I use propertree.

**Comment by scanduz** [comment]
if you can find a solution for sonoma I will gladly accept the advice, otherwise I am already satisfied with the result of this kext, which made my card work on monterey. Maybe one day support will be included natively on open core legacy patcher

**Comment by VisualEhrmanntraut** [comment, marked as answer]
as @scanduz mentioned, he's using a `MacPro5,1`. That Mac can't run Ventura and newer because Apple's kexts (not mine) require some features in the CPU that the Mac's CPU does not have.

**Comment by scanduz** [comment]
with the rx 560 I was able to install sonoma easily, then I had the happy idea of ​​installing the rx 6650 xt, I understand that the problem was with the CPU, but because with the 560 everything worked normally

**Reply by VisualEhrmanntraut** [reply]
the 560 and the 6650 use different kexts

**Reply by scanduz** [reply]
yes, in fact, all I had to do was use OCLP which installed everything. My question is whether in the future there will be the possibility of using this card on mac os ventura/sonoma

**Reply by VisualEhrmanntraut** [reply]
That has to do with your computer, not the card itself. This is not our responsibility, we don't care about legacy hardware and the ChefKiss team is compromised of 2 people. Be reasonable.

**Reply by scanduz** [reply]
I would like to ask the OCLP developers to extend support for the RX 6650 XT on Mac OS Sonoma but I don't have access to the dialogue

**Reply by VisualEhrmanntraut** [reply]
I don't think they really care.

**Reply by scanduz** [reply]
Thank you for taking the time to answer my questions🙏

**Comment by scanduz** [comment]
I'm happy to have obtained support on Mac OS Monterey thanks to nootrx and I really appreciate the work that the producers have done, helping inexperienced people like me, but I leave open the hope for the extension of support.
